ethcore/light/src/on_demand/request.rs

```diff
@@ -701,6 +739,56 @@ impl HeaderProof {
 	}
 }
 
+/// Request for a header by hash with a range of ancestors.
+#[derive(Debug, Clone, PartialEq, Eq)]
+pub struct HeaderWithAncestors(pub Field<H256>, pub u64);
```

Comment:
I would prefer a `normal struct` instead of a `tuple struct` here because it will make easier to read/understand!